Automattic/wp-calypso
# Login Social First: Change header copy

We are changing the Login Social First header from `Log into WordPress.com` to `Log in to WordPress.com`.

## Testing

1. Live link
2. Visit `/log-in`
3. Check that the header is correct
